Services/Calendar/classes/class.ilDatePresentation.php

```diff
@@ -166,6 +165,69 @@ public static function formatDate(ilDateTime $date)
 				return $date_str.', '.$date->get(IL_CAL_FKT_DATE,'g:ia',$ilUser->getTimeZone());
 		}
 	}
+
+	/**
+	 * Format a date
+	 * @access public
+	 * @param ilDate|ilDateTime $date
+	 * @return string date presentation in user specific timezone and language. This will add the date even if it is yesterday, today, or tomorrow.
+	 * @static
+	 */
+	public static function formatDate2(ilDateTime $date)
```

Comment:
@chfsx This PR is scheduled on the JF, I will get back to it after the discussion.